apps/coordinator/package.json

```diff
@@ -24,7 +24,7 @@
     "start:prod": "pnpm run run:node build/ts/main.js",
     "test": "jest --testPathIgnorePatterns=e2e.deploy.test.ts --testPathIgnorePatterns=e2e.test.ts --forceExit",
     "test:coverage": "jest --coverage --forceExit",
-    "test:e2e": "jest ./tests/*.test.ts --forceExit",
+    "test:e2e": "jest ./tests/*.test.ts --runInBand",
```

Comment:
Why did you change it?

Reply:
Because the e2e.aa.test.ts were running in parallel and we  needed the `--runInBand` flag. I forgot to readd the `--forceExit` one, I just pushed it with the new changes